### DISCUSSION TITLE
"I/O error: File not found" for Hillshades download

### DISCUSSION THREAD

**Discussion by Jadebead** [osmandapp/OsmAnd#18645] [Q&A] [answered]
I have the paid app Osmand+ and the Contour lines plugin but I can't download Hillshades, Slopes or the Terrain map for any country, on either of my phones running different versions of Android and Osmand+. I get "I/O error: File not found" when I click to download. I can download all other maps including Contour lines. How can I fix this issue? Or barring that, is there a ftp site where I could download the obf files myself?

I contacted Osmand support who suggested turning off my VPN (I don't have one), checking that Proxy was off in the Osmand app's settings (it was) and reinstalling (I did). None of that worked, and now Osmand support won't answer my emails requesting further assistance.

What could be the issue causing the I/O error? I found someone else who had the same problem as me recently:

https://github.com/osmandapp/OsmAnd/discussions/18297

**Comment by Jadebead** [comment]
Tried downloading Hillshades for my country just now and it finally worked. That's after 2-3 days of trying. There's definitely something buggy at the server end about downloading these files.

**Reply by vshcherb** [reply, marked as answer]
The issue is fixed

**Reply by Jadebead** [reply]
Thanks.
